### ISSUE TITLE
[Accessibility] [Narrator] Combo-Box control label readout incorrectly when required is true

### ISSUE THREAD

**Issue by aniketkhorjuvekar** [OfficeDev/office-ui-fabric-react#8174]
### Environment Information 
 
- __Package version(s)__: 6.146.1
- __Browser and OS versions__: Chrome 72.0.3626.119

### Please provide a reproduction of the bug in a codepen:
 

Code: https://codepen.io/aniketkhorjuvekar/pen/JzKvzm?&editable=true
Narrator Used: NVDA

#### Actual behavior: 
Narrator transcript:
ComboBox with toggleable freeform/auto-complete *  combo box  collapsed  editable
 
#### Expected behavior:
 
Narrator transcript:
ComboBox with toggleable freeform/auto-complete  combo box  collapsed  editable required

'*' should be readout as "required" 

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

**Comment by aditima** [comment]
@aftab-hassan - can we try using `aria-required` attribute and see if it helps? https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-required

**Comment by aftab-hassan** [comment]
@aditima I understand that the suggestion is to use the `aria-required` attribute, instead of relying on the `Label` text to say "required", which probably results in the issue. I'll need to dig in a  bit, but makes sense to me.

**Comment by KevinTCoughlin** [comment]
Before we proceed, have we confirmed whether or not this _incorrect_ behavior is exhibited in Edge + Narrator? We primarily target Edge + Narrator integration and need to consider whether or not this is a bug with NVDA or not.

**Comment by aftab-hassan** [comment]
Was able to reproduce this bug in both, Narrator running on Edge and NVDA running on Google Chrome. Raised a `PR` above.
